spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
associationを活用してこの記載を省いてみましょう！
```suggestion
  let(:task) { create(:task) }
```